middlewares/validators/challenges.js

```diff
@@ -17,6 +17,22 @@ const createChallenge = async (req, res, next) => {
   }
 };
 
+const subscribeToChallenge = async (req, res, next) => {
+  const schema = joi.object().strict().keys({
+    userId: joi.string().guid({ version: 'uuidv4' }).required(),
```

Comment:
Can you please explain once more why you used `version: 'uuidv4'`??
cc: @heyrandhir @RitikJaiswal75